src/feature/nodelist/routerstatus_st.h

```diff
@@ -20,6 +21,8 @@ struct routerstatus_t {
   /** Digest of the router's most recent descriptor or microdescriptor.
    * If it's a descriptor, we only use the first DIGEST_LEN bytes. */
   char descriptor_digest[DIGEST256_LEN];
+  ed25519_public_key_t *ed25519_signing_key; /**< The ed25519 signing key of
```

Comment:
Can we possibly avoid adding this field? It's stored in microdescriptors right now, not routerstatuses at all.

The reason we shouldn't add it here is that we can't ensure that this field is actually set in every routerstatus_t for a relay with an ed25519 key.  We set it in dirauth_set_routerstatus_from_routerinfo, but note that we can't set it at all when we receive the routerstatus_t from a vote.